plugins/woocommerce-admin/client/marketplace/components/product-card/product-card.scss

```diff
@@ -181,6 +192,10 @@
 			text-decoration: none;
 		}
 
+		&__sponsored-label {
+			color: $gray-700;
+		}
```

Comment:
This could maybe have been `woocommerce-marketplace__product-card__label`, as we may show labels with a different text at some point. Not a big deal I guess.